### ISSUE TITLE
Selecting text in a web page causes controls not to be announced

### ISSUE THREAD

**Issue by mitchellevan** [FreedomScientific/VFO-standards-support#702]
### Summary

1. Start JAWS, then start Google Chrome browser.
1. Go to https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-explicitly
2. Press CTRL + SHIFT + RIGHT ARROW to select the first word of the "Associating labels explicitly" heading.
3. Press TAB once to move focus to the next tab-focusable element after the selection, in this case the "First name" field. JAWS announces the "First name" field as expected.
4. Press TAB once to move focus to the next tab-focusable element after the "First name" field, in this case the "Subscribe to newsletter" checkbox.
5. Press SPACE to select or activate the focused element.

### Expected result

At step 4, visual focus should move to the "Subscribe to newsletter" checkbox, and JAWS should announce this checkbox.

At step 5, the checkbox should become checked.

### Actual result

At step 4, visual focus moves to the checkbox, but JAWS repeats the announcement of the "First name" field.

At step 5, a space character becomes entered in the "First name" field, and visual focus jumps back to the "First name" field.

This problem is not specific to forms. Selecting text in web pages then moving focus causes numerous inconsistencies like this.

### Additional Information

The problem does not occur in JAWS with Firefox.

The problem does not occur in NVDA with Chrome.

#### JAWS version and build number

JAWS 2023.2302.15
 
#### Operating System and version

Windows 10

#### Browser and version:

Chrome 110.0.5481.178

**Comment by JAWS-test** [comment]
Can confirm the problem.

No matter which text I select: when I navigate through the page with TAB afterwards, the input fields are output correctly, but other controls are not. This applies e.g. to checkboxes and links. For example, if I have the following on the page:
* First input field
* Checkox
* Second input field
* Link

then the checkbox is output as the first input field and the link as the second input field

**Comment by JAWS-test** [comment]
I think this is a serious problem that should be fixed soon!
